### PR TITLE
:seedling: rename placement decision tracker get function

### PR DESCRIPTION
**Pull request by haoqing0110**
## Summary

Rename Get() to GetClusterChanges() and add function UpdateClusters() to update the existing clusters.

## Related issue(s)

Fixes #

### CODE REVIEW

File: cluster/v1beta1/helpers.go [hunk 1]
```diff
@@ -59,13 +59,13 @@ func NewPlacementDecisionClustersTrackerWithGroups(placement *Placement, pdl Pla
 	return pdct
 }
 
-// Get updates the tracker's decisionClusters and returns added and deleted cluster names.
-func (pdct *PlacementDecisionClustersTracker) Get() (sets.Set[string], sets.Set[string], error) {
+// UpdateClusters updates the tracker's decisionClusters.
+func (pdct *PlacementDecisionClustersTracker) UpdateClusters() error {
```
Comment (qiujian16) [inline cluster/v1beta1/helpers.go]: Will Update or Refresh better?

Comment (haoqing0110) [reply cluster/v1beta1/helpers.go]: Rename to refresh, sounds better.

**Pull request merged**